README.md

```diff
@@ -105,12 +105,35 @@ For this reason, you can place a `.shared` file inside the `.githooks` repositor
 
 ```shell
 $ git config --global --get githooks.shared
-git@github.com:shared/hooks-python.git,git@github.com:shared/hooks-maven.git
```

Comment:
Are these actually unsupported now?

Reply:
Yes, I dont understand these urls? When I check with Git Url Doc I cannot find an example?

Reply:
I think this is the ssh clone URL you get from GitHub, is it not?

Reply:
You are right:
![image](https://user-images.githubusercontent.com/647437/93202242-acff1a80-f752-11ea-9b89-bdeb5cfe9c43.png)
Hm, this fact complicats matters... Have to think if we can really handle this in the code properly...
Do you know how to split them from local paths and proper urls?

Reply:
I can just check for `.+:.+` maybe, need to take care about the @<tag> syntax we introduced

Reply:
Yeah, not sure yet what's the best way.
Have a bunch of tests to run through it? 😁
`:` might be a weak test as it would be in the front as well for things like https://
Also the branch specifier might be interesting with these like git@github.com:some/repo.git@develop

Reply:
I think we first test for protocol, if its not there, we test for ".+:.+", if it matches we brutforce treat it as scp syntax...
of course taking into account the branch specifier...

Reply:
This is supported now.

Reply:
Supported now.